main/templates/auth/auth.html

```diff
@@ -24,12 +24,13 @@
 
         # set is_icon = config.CONFIG_DB.has_email_authentication
         <div class="text-center">
-          {{utils.signin_button('Google', 'btn-google', 'fa-google-plus', google_signin_url, is_icon)}}
+          {{utils.signin_button('GAE', 'btn-google', 'fa-google', gae_signin_url, is_icon)}}
```

Comment:
Even though internally it is GAE I don't really like it GAE here, because noone will know that except the devs.. maybe just `Google` and the other one `Google+` for the labels?